app.py

```diff
@@ -0,0 +1,213 @@
+        cdk.Tags.of(cdk_app).add(key, value)
+
+network_stack = NetworkStack(
+    scope=cdk_app,
+    construct_id=f"{stack_name_prefix}-network",
+    vpc_cidr=environment_variables["VPC_CIDR"],
+)
+
+ecs_stack = EcsStack(
+    scope=cdk_app,
+    construct_id=f"{stack_name_prefix}-ecs",
+    vpc=network_stack.vpc,
+    namespace=environment_variables["FQDN"],
+)
+
+# From AWS docs https://docs.aws.amazon.com/AmazonECS/latest/developerguide/service-connect-concepts-deploy.html
+# The public discovery and reachability should be created last by AWS CloudFormation, including the frontend
+# client service. The services need to be created in this order to prevent an time period when the frontend
+# client service is running and available the public, but a backend isn't.
+load_balancer_stack = LoadBalancerStack(
+    scope=cdk_app,
+    construct_id=f"{stack_name_prefix}-load-balancer",
+    vpc=network_stack.vpc,
+)
+
+api_docs_props = ServiceProps(
+    container_name="agora-api-docs",
+    container_location=f"ghcr.io/sage-bionetworks/agora-api-docs:{agora_version}",
+    container_port=8010,
+    container_memory=200,
+    container_env_vars={"PORT": "8010"},
+)
+api_docs_stack = ServiceStack(
+    scope=cdk_app,
+    construct_id=f"{stack_name_prefix}-api-docs",
+    vpc=network_stack.vpc,
+    cluster=ecs_stack.cluster,
+    props=api_docs_props,
+)
+
+mongo_props = ServiceProps(
+    container_name="agora-mongo",
+    container_location=f"ghcr.io/sage-bionetworks/agora-mongo:{agora_version}",
+    container_port=27017,
+    container_memory=500,
+    container_env_vars={
+        "MONGO_INITDB_ROOT_USERNAME": "root",
+        "MONGO_INITDB_ROOT_PASSWORD": "changeme",
+        "MONGO_INITDB_DATABASE": "agora",
+    },
+    container_volumes=[
+        ContainerVolume(
+            path="/data/db",
+            size=30,
+        )
+    ],
+)
+mongo_stack = ServiceStack(
+    scope=cdk_app,
+    construct_id=f"{stack_name_prefix}-mongo",
+    vpc=network_stack.vpc,
+    cluster=ecs_stack.cluster,
+    props=mongo_props,
+)
+
+# It is probably not appropriate host this container in ECS
+# data_props = ServiceProps(
+#     container_name="agora-data",
+#     container_location=f"ghcr.io/sage-bionetworks/agora-data:{agora_version}",
+#     container_port=9999,  # Not used
+#     container_memory=2048,
+# )
+# data_stack = ServiceStack(
+#     scope=cdk_app,
+#     construct_id=f"{stack_name_prefix}-data",
+#     vpc=network_stack.vpc,
+#     cluster=ecs_stack.cluster,
+#     props=data_props,
+#     container_env_vars={
+#         "DB_USER": "root",
+#         "DB_PASS": "changeme",
+#         "DB_NAME": "agora",
+#         "DB_PORT": "27017",
+#         "DB_HOST": "agora-mongo",
+#         "DATA_FILE": "syn13363290",
+#         "DATA_VERSION": "68",
+#         "TEAM_IMAGES_ID": "syn12861877",
+#         "SYNAPSE_AUTH_TOKEN": "agora-service-user-pat-here",
+#     },
+# )
+# data_stack.add_dependency(mongo_stack)
```

Comment:
What is the appropriate solution for deploying this out; Are there follow-up tickets to evaluate alternatives?

Reply:
I have no idea, that's probably a good question for @tschaffter, @sagely1 or @JessterB

Reply:
The commented solution would be the fastest way to get the stack deployed and working to make sure that all the components fit together. Sensitive parameters would be stored in AWS Parameter Store.

As Khai suggested, we could replace this self-managed solution with a managed solution (Amazon DocumentDB) but this extr work would need to be motivated (cost, security, maintenance, availability, throughput, etc.).

For additional context, the Agora DB is read-only from the application perspective and would be flashed with new data whenever a new data release is available. The database does not contain any sensitive information.

For this ticket, I suggest we move forward with the containerized DB and iterate on this component in a follow up ticket. Additional work is still needed to identify the SOP for pushing new data releases to the database.